compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -594,7 +594,8 @@ object Types {
             // We need to strip `JavaNull` from both the type and the prefix so that
             // `pre <: tp` continues to hold.
             tp.stripJavaNull.findMember(name, pre.stripJavaNull, required, excluded)
-          } else {
+          }
+          else {
```

Comment:
Ditto: why do we prefer this?

Reply:
Martin mentioned in the comments that putting `else` in a newline is the dotty code style

Reply:
Ok then